src/main/java/com/codeborne/selenide/ElementsCollection.java

```diff
@@ -83,9 +83,8 @@ public ElementsCollection shouldHaveSize(int expectedSize) {
   /**
    * Check if a collection matches given condition(s).
    * <p> For example: </p>
-   * {@code $$(".text_list").should(containTexts("text1", "text2"));}
-   * {@code $$(".cat_list").should(allMatch("value==cat",
-   * el -> el.getAttribute("value").equals("cat")));}
+   * {@code $$("#text_list").should(containTexts("text1", "text2"));}
```

Comment:
I totally understand that potentially it's too late, but anyway, imho based on existing selenide naming it should be named as `containExactTexts`

Reply:
@fokinp  good point!

Reply:
Yes, it sounds absolutely reasonable. 

I changed the name to `containExactTextsCaseSensitive` for the main methods, and everywhere else in the dependent code. 
Now the result code should look like this:
`$$(".list").should(containExactTextsCaseSensitive("text1", "text2"));`

Reply:
I'm just worried a bit that this change breaks existing contract and we should additionally notify all users :)

Reply:
... and that the new version will follow soon =)